.github/workflows/nightly-tier1.yml

```diff
@@ -24,6 +24,6 @@ jobs:
       - run: HUB_BASE_URL="http://$(minikube ip)/hub" DEBUG=1 make test-tier1
       - run: HUB_BASE_URL="http://$(minikube ip)/hub" make test-jira
         env:
-          JIRA_USERNAME: ${{ secrets.JIRA_USERNAME }}
-          JIRA_PASSWORD: ${{ secrets.JIRA_PASSWORD }}
-          JIRA_URL: ${{ secrets.JIRA_URL }}
+          JIRA_USERNAME: ${{ secrets.JIRA_CLOUD_USERNAME }}
```

Comment:
Rename JIRA_USERNAME to JIRA_CLOUD_USERNAME

Reply:
Thanks! Done in c7aa17a